core/chaincode/lifecycle/custodian.go

```diff
@@ -88,17 +102,25 @@ func (cc *ChaincodeCustodian) Work(buildRegistry *container.BuildRegistry, build
 			if err := launcher.Launch(chore.chaincodeID); err != nil {
 				logger.Warningf("could not launch chaincode '%s': %s", chore.chaincodeID, err)
 			}
-		} else {
-			buildStatus, ok := buildRegistry.BuildStatus(chore.chaincodeID)
-			if ok {
-				logger.Debugf("skipping build of chaincode '%s' as it is already in progress", chore.chaincodeID)
-				continue
-			}
-			err := builder.Build(chore.chaincodeID)
-			if err != nil {
-				logger.Warningf("could not build chaincode '%s': %s", chore.chaincodeID, err)
+			continue
+		}
+
+		if chore.stoppable {
```

Comment:
if // continue, if // continue // else looks like a switch.

Not something you need to address but I think this code could use a little TLC.